tests/ctst/common/common.ts

```diff
@@ -8,6 +8,51 @@ import assert from 'assert';
 
 setDefaultTimeout(Constants.DEFAULT_TIMEOUT);
 
+type listingObject = {
+    Key: string,
+    VersionId: string,
+}
+
+type listingResult = {
+    Versions: listingObject[],
+    DeleteMarkers: listingObject[],
+}
+
+/**
+ * Cleans the created test bucket
+ * @param {Zenko} world world object
+ * @param {string} bucketName bucket name
+ * @returns {void}
+ */
+export async function cleanZenkoBucket(
```

Comment:
not really a "Zenko" bucket: this would actually work for any S3 bucket...
the only link to Zenko is the type of `world`, and I am pretty sure this may be replaced with a more generic type when needed.